components/batches/batches.tsx

```diff
@@ -30,6 +30,7 @@ const Batch: FC<BatchProps> = ({ batch, projects }) => {
         {projects
           .filter(
             ({ batches }) =>
+              batches["filter"] &&
```

Comment:
Is needed as when we create a new project batches is a function and not an array. This is due to the subscription ignoring the selection set.